src/Microsoft.Azure.WebJobs.Protocols/FunctionFailure.cs

```diff
@@ -33,7 +33,7 @@ internal static FunctionFailure FromException(Exception ex)
             {
                 Exception = ex,
                 ExceptionType = ex.GetType().FullName,
-                ExceptionDetails = ex.ToDetails()
+                ExceptionDetails = ex is ObjectDisposedException ? ex.ToString() : ex.ToDetails() // Do not format ObjectDisposedExceptions to aid in debugging
```

Comment:
Should we do this inside the `ToDetails`? I wonder folks will miss doing this conditional check in future when they have to use this exception type in some new code they are writing.